Added feedback after partner event (#558)
AdobeDocs/commerce-extensibility: src/pages/app-management/configuration-schema.md

| `description` | string | No | Help text displayed below the field. |
| `options` | array | Conditional | Required for `list`. Defines available options to be displayed in the dropdown list. |
| `selectionMode` | string | Conditional | Required for `list`. Set to `single` for standard dropdown or `multiple` to allow multiple selections. |

> | `env` | array | No | Limits the field to **PaaS** (`"paas"`), **SaaS** (`"saas"`), or both. When omitted, the field applies to all environments. See [Commerce environment fields](#commerce-environment-fields). |

Reviewer: Weird behavior.

```suggestion
| `env` | array | No | Limits the field to **PaaS** (`paas`) or **SaaS** (`saas`). To enable the field to all environments, omit the field or specify both values. See [Commerce environment fields](#commerce-environment-fields). |
```

| `url` | string | URL input with validation |
| `list` | string | Dropdown with preconfigured options |

> ### Commerce environment fields

Reviewer: There's only one field and you explained it above. I don't think you're clarifying anything by duplicating the info.

I don't think you should be linking to repo READMEs. The READMEs should be deleted, with the docs being the sole source of truth.

Before you run `getConfiguration`, `getConfigurationByKey`, or `setConfiguration`, call `initialize` with your generated configuration schema. The schema is kept in memory for that action invocation. Import the generated `configuration-schema.json` from your `commerce/configuration/1` extension (the path relative to your action depends on your project layout).

If you skip `initialize`, see [Initialization](https://github.com/adobe/aio-commerce-sdk/blob/main/packages/aio-commerce-lib-config/docs/usage.md#initialization) in the configuration library usage guide for more information.

A **scope selector** tells the library which node in the scope tree to read or write. That tree can include **Adobe Commerce** scopes (such as websites and store views, each with a scope code and a **level** in the hierarchy), **custom scopes** you create in App Management (code only; see below), **global** scope, and other nodes that your app or merchants configure.

When the target scope has both a code and a level, which is typical for Commerce store and website scopes, use `byCodeAndLevel`.

> The following examples show only the configuration calls. In your action, run `initialize` before these calls, as described in the opening paragraphs of this section. For a full runtime action example, see [Using configuration in runtime actions](https://github.com/adobe/aio-commerce-sdk/blob/main/packages/aio-commerce-lib-config/docs/usage.md#using-configuration-in-runtime-actions) in the usage guide.

Reviewer 2: ```suggestion
The following examples show only the configuration calls. In your action, run `initialize` before these calls. For a full runtime action example, see [Using configuration in runtime actions](https://github.com/adobe/aio-commerce-sdk/blob/main/packages/aio-commerce-lib-config/docs/usage.md#using-configuration-in-runtime-actions) in the usage guide.
```

Reviewer 2: Can you link to where it's described instead?

Use `getConfigurationByKey` from the configuration library (together with `getConfiguration` and `setConfiguration` when you need full documents or writes) to access configuration values in your runtime actions.

Before you run `getConfiguration`, `getConfigurationByKey`, or `setConfiguration`, call `initialize` with your generated configuration schema. The schema is kept in memory for that action invocation. Import the generated `configuration-schema.json` from your `commerce/configuration/1` extension (the path relative to your action depends on your project layout).

If you skip `initialize`, see [Initialization](https://github.com/adobe/aio-commerce-sdk/blob/main/packages/aio-commerce-lib-config/docs/usage.md#initialization) in the configuration library usage guide for more information.

A **scope selector** tells the library which node in the scope tree to read or write. That tree can include **Adobe Commerce** scopes (such as websites and store views, each with a scope code and a **level** in the hierarchy), **custom scopes** you create in App Management (code only; see below), **global** scope, and other nodes that your app or merchants configure.

> When the target scope has both a code and a level—typical for Commerce store and website scopes—use `byCodeAndLevel`.

Reviewer: ```suggestion
When the target scope has both a code and a level, which is typical for Commerce store and website scopes, use `byCodeAndLevel`.
```
